files/image_config/procdockerstats/procdockerstats

```diff
@@ -0,0 +1,210 @@
+    syslog.syslog(syslog.LOG_INFO, msg)
+    syslog.closelog()
+
+def log_warning(msg, also_print_to_console=False):
+    syslog.openlog(SYSLOG_IDENTIFIER)
+    syslog.syslog(syslog.LOG_WARNING, msg)
+    syslog.closelog()
+
+def log_error(msg, also_print_to_console=False):
+    syslog.openlog(SYSLOG_IDENTIFIER)
+    syslog.syslog(syslog.LOG_ERR, msg)
+    syslog.closelog()
+
+# ========================== ProcessDocker class ==========================
+class ProcDockerStats:
+
+    def __init__(self):
+        self.state_db = swsssdk.SonicV2Connector(host=REDIS_HOSTIP)
+        self.state_db.connect("STATE_DB") 
+    
+    def run_command(self, cmd):
+        proc = subprocess.Popen(cmd, shell=True, stdout=subprocess.PIPE)
+        (stdout, stderr) = proc.communicate()
+        if proc.returncode != 0:
+        log_error("Error running command '{}'".format(cmd))
+        else:
+            return stdout
+
+    def format_docker_cmd_output(self, cmdout):
+        lines = re.split("\n", cmdout)
+        keys = re.split("   +", lines[0])
+        dict1 = dict()
+        dict_list = []
+        for item in lines[1:]:
+            values1 = re.split("   +", item)
+            dict1 =  dict(zip(keys, values1))
+            dict_list.append(dict1)
+        formatted_dict = self.create_docker_dict(dict_list)
+        return formatted_dict
+
+    def format_process_cmd_output(self, cmdout):
+        lines = re.split("\n", cmdout)
+        keys = re.split(" +", lines[0])
+        keylist = list(filter(None, keys))
+        dict1 = dict()
+        dict_list = []
+        for item in lines[1:]:
+            values1 = re.split(" +", str(item))
+            # To remove extra space before UID
+            val = list(filter(None, values1))
+            # Merging extra columns created due to space in cmd ouput
+            val[8:] = [''.join(val[8:])] 
+            dict1 = dict(zip(keylist, val))
+            dict_list.append(dict1)
+        return dict_list
+
+    def convert_to_bytes(self, value):
+        u = re.search('[a-zA-Z]+', value)
+        v = float(filter(str.isdigit, value))
+        unit = u.group(0)
+        UNITS_B = 'B'
+        UNITS_KB = 'KB'
+        UNITS_MB = 'MB'
+        UNITS_MiB = 'MiB'
+        UNITS_GiB = 'GiB'
+        if unit.lower() == UNITS_B.lower():
+            return v
+        elif unit.lower() == UNITS_KB.lower():
+            v1 = v * 1000
+            return v1
+        elif unit.lower() == UNITS_MB.lower():
+            v1 = v * 1000 * 1000
+            return v1
+        elif unit.lower() == UNITS_MiB.lower():
+            v1 = v * 1024 * 1024
+            return v1
+        elif unit.lower() == UNITS_GiB.lower():
+            v1 = v * 1024 * 1024 * 1024
+            return round(v1)
+
+    def create_docker_dict(self, dict_list):
+        dockerdict = {}
+        for row in dict_list[0:]:
+            cid = row.get('CONTAINER ID')
+            if cid:
+                key = 'Docker_Stats|' + str(cid) 
```

Comment:
Naming convention for SONiC DB tables is UPPERCASE. Please change to `DOCKER_STATS|` here, and do the same for all table names in this file?